CrewChiefV4/Audio/AudioPlayer.cs

```diff
@@ -1008,15 +1008,15 @@ public Boolean isChannelOpen()
             return channelOpen;
         }
 
-        public void playMessage(QueuedMessage queuedMessage)
+        public void playMessage(QueuedMessage queuedMessage, int priority = SoundMetadata.DEFAULT_PRIORITY)
         {
             if (GlobalBehaviourSettings.enabledMessageTypes.Contains(MessageTypes.NONE))
             {
                 Console.WriteLine("All messages disabled for this car class. Message " + queuedMessage.messageName + " will not be played");
             } 
             else
             {
-                playMessage(queuedMessage, PearlsOfWisdom.PearlType.NONE, 0);
+                playMessage(queuedMessage, PearlsOfWisdom.PearlType.NONE, 0, priority);
```

Comment:
should there be a validation of priority range?  Also, I think priority out of set of constants might be more readable in code.

Reply:
yeah... I did think about that but I wanted to be able to make it fine-grained. Maybe once this feature settles down I'll revisit it